### PR TITLE
finish match pages

### LINKED ISSUE #197: Finish match pages
- [x] ward map
- [x] headings for tables/sections

### PR DESCRIPTION
**Pull request by howardchung**
fixes #197 

Could use some more styling on vision map.

Reviewers:
@albertcui 
@nicholashh

### CODE REVIEW

File: src/components/Table/Table2.jsx [hunk 1]
```diff
@@ -21,21 +21,28 @@ import styles from './Table.css';
 class Table extends React.Component {
   componentWillMount() {
     this.setState({
-      data: this.props.data,
       sortState: undefined,
+      sortField: undefined,
```
Comment (dartharaknus) [inline src/components/Table/Table2.jsx]: Don't need this file.

File: src/components/Match/VisionMap.jsx [hunk 2]
```diff
@@ -25,44 +41,111 @@ const senWard = (style, iconSize) => (<svg style={style} width={iconSize} height
...
+          }}
+        >
+          <img width={this.props.width} src="/assets/images/map.png" role="presentation" />
+          {obsIcons}
+          {senIcons}
+        </div>
+      </Col>
+      <Col md={6}>
+        {
+        /*
```
Comment (dartharaknus) [inline src/components/Match/VisionMap.jsx]: What is this part?

File: src/components/Table/Table2.jsx (hunk 1 above)
Comment (howardchung) [inline src/components/Table/Table2.jsx]: removed

File: src/components/Match/VisionMap.jsx (hunk 2 above)
Comment (howardchung) [inline src/components/Match/VisionMap.jsx]: old style selector.  I think the new one is better, but keeping this around for easy swapping until we finalize.

**Pull request merged**